lib/core/diagnostics.js

```diff
@@ -111,6 +117,64 @@ if (undiciDebugLog.enabled || fetchDebuglog.enabled) {
     )
   })
 
+  // Track fecth requests
+  diagnosticsChannel.channel('undici:fetch:start').subscribe(evt => {
+    const {
+      context: { input }
+    } = evt
```

Comment:
It seems that the fields you want are not wrapped in context?!

```suggestion
    const {
      input
    } = evt
```